src/v/kafka/server/tests/create_topics_test.cc

```diff
@@ -24,7 +24,8 @@
 class create_topic_fixture : public redpanda_thread_fixture {
 public:
     kafka::create_topics_request make_req(
-      std::vector<kafka::creatable_topic> topics, bool validate_only = false) {
+      chunked_vector<kafka::creatable_topic> topics,
```

Comment:
The edits in this file are because chunked_vector does not have a copy constructor, so we need to move the request object around now.

This means copying some stuff out here where we used the request after passing it down, and more use of `std::move`.